### PR TITLE
fix: Sort workspace by name by created_at

### PR DESCRIPTION
**Pull request by Emyrk**

### CODE REVIEW

File: coderd/database/queries/workspaces.sql [hunk 1]
```diff
@@ -70,7 +70,8 @@ FROM
 WHERE
 	owner_id = @owner_id
 	AND deleted = @deleted
-	AND LOWER("name") = LOWER(@name);
+	AND LOWER("name") = LOWER(@name)
```
Comment (Kira-Pilot) [inline coderd/database/queries/workspaces.sql:73]: 😅

**Review by Kira-Pilot — APPROVED** [review]
sweet

File: coderd/database/queries/workspaces.sql (hunk 1 above)
Comment (Emyrk) [reply coderd/database/queries/workspaces.sql:73]: Wait I'm confused what this means :thinking:

Comment (Kira-Pilot) [reply coderd/database/queries/workspaces.sql:73]: oh, sorry @Emyrk, this wasn't actionable and was actually directed at the sorting line below.

**Pull request merged**